src/main/frontend/extensions/sci.cljs

```diff
@@ -9,27 +9,33 @@
   (/ (reduce + coll) (count coll)))
 
 (defn eval-string
-  [s]
+  ([s]
+   (eval-string s {}))
+  ([s ns]
   (try
     (sci/eval-string s {:bindings {'sum sum
                                    'average average
                                    'parseFloat js/parseFloat
+                                   'custom-js (if (exists? js/customJs) js/customJs (Empty.))
```

Comment:
This workflow seems confusing as I wouldn't expect custom.js fns to be reusable in cljs contexts. I think a separate issue explaining a use case would help. We could possibly brainstorm a more intuitive solution

Reply:
example usage custom-js function in query `:view`
![image](https://user-images.githubusercontent.com/7157397/202111172-5d8af75a-c38b-4db2-9067-f25ce6b29790.png)

Reply:
more example use case. from custom view of query, I can toggle property value or use text input to update property from query view

https://user-images.githubusercontent.com/7157397/202222733-3ff29a18-3f47-4bfa-b5a7-5e02c5421831.mp4

Reply:
Those examples are helpful. It seems you want to build a library of personal fns and have them available in sci. I think this could be valuable but I think the current solution is arbitrary. Other users may want different names than `customJs` or may want to define their fns as cljs. I think this workflow needs more thought and would recommend a separate issue for it